frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -135,6 +136,22 @@ const plugin: Plugin<ConsumedExtensions> = [
       disallowed: [OCS_INDEPENDENT_FLAG],
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: true,
+      path: [
+        `/k8s/ns/:ns/${referenceForModel(
+          ClusterServiceVersionModel,
+        )}/:appName/${storageSystemGvk}/~new`,
+        `/k8s/ns/:ns/${ClusterServiceVersionModel.plural}/:appName/${storageSystemGvk}/~new`,
+      ],
+      loader: () =>
+        import(
+          './components/create-storage-system/create-storage-system' /* webpackChunkName: "create-storage-system" */
+        ).then((m) => m.default),
+    },
+  },
```

Comment:
Use `console-extensions.json`

Reply:
IIRC, you mentioned routes have issues with dynamic plugins.
I have not done those changes therefore.

Reply:
my bad, you're overriding the OLM page. And yeah console-extensions is not going to cut it.